From 0604c16f38f423fb3157179fd5783f4e772908e9 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 21 Nov 2018 16:42:12 +0200
Subject: [PATCH 5180/5242] MLK-20328-8: ARM64: dtsi: mx8mq: adjust the
 expected format

commit  2e1e8f2dd4df35dc08eb3a50f69ba16a36032a8e from
https://source.codeaurora.org/external/imx/linux-imx.git

Default value must be prefixed by "0".

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit 1a8e10215cc07c83b0e26b55fef94aae18151633)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 7fa00f6..f4dcf7a 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -1035,7 +1035,7 @@
 		clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3";
 		dmas = <&sdma2 8 2 0>, <&sdma2 9 2 0>;
 		dma-names = "rx", "tx";
-		fsl,dataline = <0xff 0xff>;
+		fsl,dataline = <0 0xff 0xff>;
 		status = "disabled";
 	};
 
@@ -1068,7 +1068,7 @@
 		dmas = <&sdma2 2 24 0>, <&sdma2 3 24 0>;
 		dma-names = "rx", "tx";
 		fsl,shared-interrupt;
-		fsl,dataline = <0xf 0xf>;
+		fsl,dataline = <0 0xf 0xf>;
 		status = "disabled";
 	};
 
@@ -1084,7 +1084,7 @@
 		clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3";
 		dmas = <&sdma2 0 24 0>, <&sdma2 1 24 0>;
 		dma-names = "rx", "tx";
-		fsl,dataline = <0x0 0xf>;
+		fsl,dataline = <0 0x0 0xf>;
 		status = "disabled";
 	};
 
-- 
1.7.9.5

